packages/hoc-input/src/formInput.story.mdx

```diff
@@ -0,0 +1,412 @@
+import { Meta, Story, Preview } from "@storybook/addon-docs/blocks";
+import { action } from "@storybook/addon-actions";
+import styled from "styled-components";
+import { MdSearch } from "@crave/farmblocks-icon";
+
+import formInput from "./index";
+import { FbInput, Container, WithValue, FocusContainer } from "./formInput.story.utils";
+
+<Meta title="Form|HOC formInput" />
+
+# hoc-input
+
+A High Order Component that adds behaviour of label support, different styles
+for focus and errors, and validation error messages.
+
+This component uses the html `input` element. So the APIs are very similar.
+
+For example, `value`, `placeholder`, `disabled`, `maxLength`, `onChange` and
+others not listed here, should work as expected, you can assume that the
+property will be forwarded to the core html input element. Ref.:
+[MDN &lt;input&gt;](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input)
```

Comment:
This text should be in the `TextInput` docs, no?

The HOC will use whichever component we pass to it.

Reply:
Hmmm yeah! It makes sense... I'm gonna move it, thanks!

Reply:
Changes addressed 😄

Reply:
Cool.
I see the text is duplicated. You moved the text in the readme, but it's still present in this mdx file.

Reply:
See again :P

Reply:
lol, you squashed the changes to the wrong commit 😆
But it's fine.